package.json

```diff
@@ -66,10 +66,12 @@
     "typeahead.js": "^0.11.1",
     "typeahead.js-browserify": "Javier-Rotelli/typeahead.js-browserify#~1.0.7",
     "urlhash": "^0.1.3",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^3.3.12",
     "woofmark": "~4.2.0"
   },
   "devDependencies": {
-    "webpack-dev-server": "^4.7.4"
+    "webpack-dev-server": "^3"
```

Comment:
Hi @illyShelly, can you remove the package.json and yarn.lock files from this PR? Thanks!

Reply:
I guess I fixed that. Just my local vs. remote branch have diverged. I tried to fixed that but not successfully yet 😅